### ISSUE TITLE
EarlyStopping in FastText

### ISSUE THREAD

**Issue by sravanbabuiitm** [facebookresearch/fastText#508]
I m interested to know what FastText suggests for dealing with overfitting ?
We can definitely add EarlyStopping to FastText, which helps in determining the ideal number of epochs during training.
I have also observed with IMDB dataset that adding n-grams with n >=4 size is leading to severe overfitting with training accuracy reaching 1, where was test accuracy remains around 0.87. Adding a variational dropout in my custom impl of FastText seemed to mitigate the problem slightly.
Is FastText planning on adding support to EarlyStopping or Dropout ?

Would it be fine for us to make changes to the code base ?

**Comment by jonDel** [comment]
Just use keras, much more friendly, supported and easy to integrate into your pipeline. See the example here: https://github.com/keras-team/keras/blob/master/examples/imdb_fasttext.py

**Comment by TheCidRMC** [comment]
Hi, the link is broken, can you provide us please the right one? Thanks
